packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -4,10 +4,8 @@
 //  * Licensed under the BSD 3-Clause license.
 //  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 //  */
+import { TestRunner } from '@salesforce/salesforcedx-utils-vscode/out/src/cli/';
 import * as events from 'events';
-import * as path from 'path';
-import * as pathExists from 'path-exists';
```

Comment:
Moving `TestRunner` to the `salesforcedx-utils-vscode` module allows us to remove `path-exists` from this module's `package.json`.

Reply:
Actually, it looks like `path-exists` is needed for the `requirements.ts` file within the `salesforcedx-vscode-apex` module.